views/layouts/main.hbs

```diff
@@ -28,7 +28,7 @@
         {{#each scripts}}
         <script src="/js/{{this}}"></script>
         {{/each}}
-        <title>{{title}} - TitaFindsXG5</title>
+        <title>{{title}} - TitaFinds</title>
```

Comment:
Must be "TitaFinds Inventory"

Reply:
see [ff9f289](https://github.com/erik-lance/TitafindsXG5/pull/11/commits/ff9f28905aaea1d93ce0ee0f4de2ade76e361ff1)